lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -776,19 +899,20 @@ def create(self, trans: GalaxyWebTransaction, payload, **kwd):
         :type __tags: list[str]
         """
         tool_id = payload.get("tool_id")
-        tool_uuid = payload.get("tool_uuid")
-        if tool_id in PROTECTED_TOOLS:
-            raise exceptions.RequestParameterInvalidException(
-                f"Cannot execute tool [{tool_id}] directly, must use alternative endpoint."
-            )
-        if tool_id is None and tool_uuid is None:
-            raise exceptions.RequestParameterInvalidException("Must specify a valid tool_id to use this endpoint.")
```

Comment:
Do we not need to validate that tool_id or tool_uuid are provided ?

Reply:
Restored.